src/routes/safe/components/Transactions/TxList/styled.tsx

```diff
@@ -474,11 +508,11 @@ export const OwnerListItem = styled.li`
   }
 `
 
-export const InlineEthHashInfo = styled(EthHashInfo)`
+export const InlinePrefixedEthHashInfo = styled(PrefixedEthHashInfo)`
```

Comment:
I would not change this but create a new one instead. This likely affects other areas.

Reply:
I had replaced it because there was only one occurrence but I am also using it now 👍